dev-haskell/stack/metadata.xml

```diff
@@ -6,20 +6,7 @@
 		<name>Gentoo Haskell</name>
 	</maintainer>
 	<use>
+		<flag name="developer-mode">By default, should extra developer information be output?</flag>
 		<flag name="integration-tests">Run the integration test suite.</flag>
```

Comment:
This line should be removed since the `integration-tests` flag is not present in `IUSE`. `repoman -x full` will throw an error because of this.